helm/prometheus-rules/templates/kaas/tenet/alerting-rules/pods.rules.yml

```diff
@@ -0,0 +1,41 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: pods.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: pods
+    rules:
+    # PodsUnschedulable fires when too many pods are in `unschedulable` status in the `kube-system` namespace
+    # This is a signal something is wrong with the WC.
+    - alert: PodsUnschedulable
+      annotations:
+        description: '{{`Cluster {{ $labels.cluster_id }} has unschedulable kube-system pods.`}}'
+        runbook_url: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/cluster
+      expr: |-
+        count(
+          count_over_time(
+            # Have a list of unschedulable pods
+            count(
+              kube_pod_status_unschedulable{namespace="kube-system"}
+            ) by (cluster_id, cluster_type, customer, installation, pipeline, provider, region, pod)
+          # only keep those that have been unschedulable for more than 5min over the past 30min
+          [30m:]) > 5 
+        # count per cluster
+        ) by (cluster_id, cluster_type, customer, installation, pipeline, provider, region)
+        > 2
+      for: 15m
+      labels:
+        area: kaas
+        # Let's start with business hours only, maybe 24x7 in the future
+        cancel_if_outside_working_hours: true
+        inhibit_cluster_broken: true
+        severity: page
+        # Let's start sending these alerts to atlas, then we can swich to provider team when properly tuned.
+        team: atlas
```

Comment:
I am not so sure about this I would rather hand this over to kaas, this is not an observability topic.

Reply:
That's the plan, but only once we're happy with how the alert behaves.

Reply:
All right, let them deal with this.
I've updated the code, but I'll need to warn them in Slack as well before merging then.